src/compile/front.rs

```diff
@@ -193,18 +187,10 @@ async fn bindgen(proj: &Project) -> Result<Outcome<Product>> {
     Ok(Outcome::Success(Product::Front))
 }
 
-async fn optimize(
-    file: &Utf8Path,
-    interrupt: broadcast::Receiver<()>,
-) -> Result<CommandResult<()>> {
-    let wasm_opt = Exe::WasmOpt.get().await.dot()?;
-
-    let args = [file.as_str(), "-Oz", "-o", file.as_str()];
-    let process = Command::new(wasm_opt)
-        .args(args)
-        .spawn()
-        .context("Could not spawn command")?;
-    wait_interruptible("wasm-opt", process, interrupt).await
+fn optimize(file: &Utf8Path) -> Result<()> {
+    OptimizationOptions::new_optimize_for_size_aggressively()
+        .run(file, file)
```

Comment:
Not sure how to make this interruptable since `run` is synchronous. Unsure how much it's needed since it's not a separate process anymore.